apm-agent-plugins/apm-api-plugin/src/test/java/co/elastic/apm/api/ElasticApmApiInstrumentationTest.java

```diff
@@ -148,13 +147,13 @@ void testAutomaticAndManualTransactions() {
     void testGetId_distributedTracingEnabled() {
 
         co.elastic.apm.agent.impl.transaction.Transaction transaction = tracer.startRootTransaction(null).withType(Transaction.TYPE_REQUEST);
-        try (Scope scope = transaction.activateInScope()) {
+        try (co.elastic.apm.agent.tracer.Scope scope = transaction.activateInScope()) {
```

Comment:
[minor] If I understand this correctly, in this case we can just change the original import and not use the FQN, and we will do the same when modifying all the plugins to use the newly created tracer API.

Reply:
*apm-agent-api* does not currently depend on the new tracer module, so I wanted to avoid the additional refactoring in the first round.